tests/quasi-newton/helpers.cpp

```diff
@@ -295,6 +295,8 @@ void runTestQNWithWaveforms(std::string const &config, TestContext const &contex
 
     interface.readData(meshName, readDataName, {vertexIDs, 2}, dt, {inValues, 2});
 
+    interface.readData(meshName, readDataName, {vertexIDs, 2}, dt, {inValuesOff, 2});
```

Comment:
@NiklasKotarsky Currently, there seems to be a problem with this test. Can you add some comments to explain the modification?